### PR TITLE
SDKS-841 Revisit Rooted Device Detection

### PR DESCRIPTION
**Pull request by witrisna**

### CODE REVIEW

File: forgerock-auth/src/androidTest/java/org/forgerock/android/auth/RootDeviceTest.java [hunk 1]
```diff
@@ -34,8 +34,9 @@ public void testDangerousPropertyDetector() {
         RootDetector rootDetector = FRRootDetector.builder()
                 .detector(new DangerousPropertyDetector())
                 .build();
+        int sdkVersion = android.os.Build.VERSION.SDK_INT;
```
Comment (rodrigoareis) [inline forgerock-auth/src/androidTest/java/org/forgerock/android/auth/RootDeviceTest.java:37]: Can be `static` and `final` at class level once it's used on multiple tests?

**Review by rodrigoareis — APPROVED** [review]
Changes looks good to me

File: forgerock-auth/src/androidTest/java/org/forgerock/android/auth/RootDeviceTest.java (hunk 1 above)
Comment (witrisna) [reply forgerock-auth/src/androidTest/java/org/forgerock/android/auth/RootDeviceTest.java]: Updated

**Review by rodrigoareis — APPROVED** [review]
Looks good to me

**Pull request merged**